sebs/benchmark.py

```diff
@@ -348,6 +348,8 @@ def install_dependencies(self, output_dir):
                 language=self.language_name,
                 runtime=self.language_version,
             )
+            # PK: To do: Marcin Need to add this docker image with fission to the dockerhub
```

Comment:
You can also change the Docker registry in `config/systems.json` to use your own Docker Hub account - for testing.